Libraries/Components/WebView/WebView.ios.js

```diff
@@ -109,7 +128,7 @@ var WebView = React.createClass({
     source: PropTypes.oneOfType([
       PropTypes.shape({
         /*
-         * The URI to load in the WebView. Can be a local or remote file.
+         * The URI to load in the web view. Can be a local or remote file.
```

Comment:
Just curious why split out "web view" instead of codifying `WebView`? Aren't all these talking about the specific instance of the `WebView` in question? Or is "web view" the more generic thing that is being discussed?